config/environment.py

```diff
@@ -65,8 +66,10 @@
     OAUTH_TOKEN = os.environ.get('OAUTH_TOKEN')
     SECRET_KEY = os.environ.get('SECRET_KEY') or 'NO SECRETS'
     HOOK_SECRET_KEY = os.environ.get('HOOK_SECRET_KEY') or 'SECRETS'
-    ANONYMOUS_REPORTING = os.environ.get('ANONYMOUS_REPORTING') or 'ON'
+    ANONYMOUS_REPORTING_ENABLED = os.environ.get('ANONYMOUS_REPORTING') or True  # noqa
 
+if ANONYMOUS_REPORTING_ENABLED in ['True', 'False']:
+    ANONYMOUS_REPORTING_ENABLED = strtobool(ANONYMOUS_REPORTING_ENABLED)
```

Comment:
Maybe a comment explaining that env variable is a string would be helpful here for the future?